### PR TITLE
Fix the `ShardBlock` fields, `data` -> `body` and  `Bytes[PLACEHOLDER]` -> `BytesN[PLACEHOLDER]`

### PR DESCRIPTION
**Pull request by hwwhww**

### CODE REVIEW

File: specs/core/1_shard-data-chains.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ class ShardBlock(Container):
     shard: Shard
     beacon_chain_root: Hash
     parent_root: Hash
-    data: ShardBlockBody
+    body: ShardBlockBody
```
Comment (vbuterin) [inline specs/core/1_shard-data-chains.md]: We should discuss just removing the one-item `ShardBlockBody` struct and making it directly be a `Bytes` during the call :smile:

Comment (djrtwo) [reply specs/core/1_shard-data-chains.md]: probably reasonable..

**Comment by djrtwo** [comment]
@hwwhww Did we make a call on the `ShardBlockBody` struct?

**Comment by hwwhww** [comment]
The shard block rework will be included in #1294.

**Pull request closed without merging**